torrentinfo.py

```diff
@@ -6,33 +6,65 @@
 import bleach
 import requests
 from lxml import etree
+import re
 
-NYAA_URL = 'https://www.nyaa.se/?page=view&tid=%s'
+NYAA_URL = 'https://nyaa.si/view/%s'
+ANIDEX_URL = 'https://anidex.info/torrent/%s'
 
-
```

Comment:
Still need two blank lines before function defs (also below for anidex function).

Reply:
Why even? Also, I thought that there are too many lines at first and didn't understand your comment.

Reply:
This is why:

> Surround top-level function and class definitions with two blank lines.
> — [PEP8, § Blank Lines](https://www.python.org/dev/peps/pep-0008/#blank-lines)

Aside from the guideline, I just think it looks better.